controllers/mdshape.py

```diff
@@ -0,0 +1,41 @@
+from bson.json_util import dumps
+from flask import Blueprint, make_response, request
+import xml.etree.ElementTree as ET
+from cerberus import Validator
+import requests
+import json
+# create the blueprint (controller) for the routes
+mdshape = Blueprint('shape', __name__)
```

Comment:
name of blueprint is 'shape', not 'mdshape'. Does the routing still work?

Reply:
it worked on local, but yeah better to change it. Good call